src/main/webapp/resources/js/pages/analysis/components/AnalysisDetails.jsx

```diff
@@ -0,0 +1,247 @@
+
+  const { analysisContext, analysisContextUpdateSubmissionName } = useContext(
+    AnalysisContext
+  );
+
+  // List of analysis details
+  const analysisDetails = [
+    {
+      title: getI18N("AnalysisDetails.id"),
+      desc: analysisContext.analysis.identifier
+    },
+    {
+      title: getI18N("AnalysisDetails.pipeline"),
+      desc: `${analysisDetailsContext.workflowName} (${analysisDetailsContext.version})`
+    },
+    {
+      title: getI18N("AnalysisDetails.priority"),
+      desc: analysisDetailsContext.priority
+    },
+    {
+      title: getI18N("AnalysisDetails.created"),
+      desc: formatDate({ date: analysisDetailsContext.createdDate })
+    },
+    {
+      title: getI18N("AnalysisDetails.duration"),
+      desc: getHumanizedDuration({ date: analysisDetailsContext.duration })
+    }
+  ];
+
+  /*
+        On change of checkbox to receive/not receive an email upon
+        pipeline completion update emailPipelineResult field
+    */
+  function updateEmailPipelineResult(e) {
+    analysisDetailsContextUpdateEmailPipelineResult(e.target.checked);
+  }
+
+  // Update analysis name
+  function updateSubmissionName(newSubmissionName) {
+    if (
+      newSubmissionName !== "" &&
+      newSubmissionName !== analysisContext.analysisName
+    ) {
+      analysisContextUpdateSubmissionName(newSubmissionName);
+    }
+  }
+
+  // Update analysis priority
+  function updateAnalysisPriority(updatedPriority) {
+    analysisDetailsContextUpdateSubmissionPriority(updatedPriority);
+  }
+
+  // Render priorities in priority dropdown (Admins only)
+  function renderPriorities() {
+    const priorityList = [];
+
+    for (let priority of analysisDetailsContext.priorities) {
+      priorityList.push(
+        <Select.Option key={priority} value={priority}>
+          {priority}
+        </Select.Option>
+      );
+    }
+    return priorityList;
+  }
+
+  /*
+   * The following renders the analysis details, and tabs
+   * for Samples, Share Results, and Delete Analysis which
+   * the components are only loaded if the corresponding
+   * tab is clicked
+   */
+  return (
+    <>
+      <Tabs
+        defaultActiveKey="analysis_details"
+        tabPosition="left"
+        style={{ marginLeft: 50, paddingTop: 25 }}
+        animated={false}
+      >
+        <TabPane
+          tab={getI18N("AnalysisDetails.details")}
+          key="analysis_details"
+          style={{ minWidth: 300 }}
+          className="t-analysis-settings-tab-details"
+        >
+          <Col span={12}>
+            <Title level={2}>{getI18N("AnalysisDetails.details")}</Title>
+            <>
+              <label>{getI18N("AnalysisDetails.name")}</label>
+              <Paragraph editable={{ onChange: updateSubmissionName }}>
+                {analysisContext.analysisName}
+              </Paragraph>
+            </>
+
+            {isAdmin && analysisContext.analysisState === "NEW" ? (
+              <>
+                <label>Priority</label>
```

Comment:
Internationalization

Reply:
Fixed in 6e737400a12193e84f2504d42dbf3a4892a58031